website/src/components/landing/sections/pi/05c-diff-grid-a.html

```
@@ -0,0 +1,87 @@
+          <p class="pi-diff-payoff">pi ships no sub-agents and no background bash, on purpose. iii ships both because a system needs them.</p>
+        </div>
+      </div>
+      <div class="pi-diff-proof">
+        <article class="hello-card">
+          <div class="hello-chead">
+            <div class="hello-meta">
+              <div class="t">reports::nightly</div>
+              <div class="s">a cron trigger, registered once</div>
+            </div>
+          </div>
+          <pre class="hello-code" data-lang="typescript"><code>worker.registerFunction("reports::nightly", async () => {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)